app/lib/widgets/patient/move_overview.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart';
+import 'package:manvsim/constants/manv_icons.dart';
+import 'package:manvsim/models/location.dart';
+import 'package:manvsim/models/patient.dart';
+
+class MoveOverview extends StatelessWidget {
+  final Patient patient;
+  final Location to;
+
+  const MoveOverview(
+      {super.key,
+        required this.to,
+        required this.patient});
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      width: double.infinity,
+      child: Padding(
+        padding: const EdgeInsets.all(16),
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.start,
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Row(children: [
+              const Icon(ManvIcons.patient),
+              const SizedBox(width: 8),
+              Expanded(
+                  child: Text(patient.name,
+                      overflow: TextOverflow.ellipsis,
+                      style: Theme.of(context).textTheme.headlineMedium)),
+            ]),
+            Row(children: [
+              const Icon(Icons.arrow_back),
```

Comment:
I think its kinda confusing having this arrow pointing in this direction here, i think it might be better to leave it out and just have the Text in this row.  
Alternatively to be consistent with the Icon + Text format, I'd use the standard location icon.